libcaliptra/test/if_test/if_test.c

```diff
@@ -0,0 +1,21 @@
+//Licensed under the Apache-2.0 license
+
+o#include <stdint.h>
```

Comment:
This doesn't compile due to this extra character

Reply:
oops, sorry about that.